src/telemetry/process.rs

```diff
@@ -11,7 +11,7 @@ metrics! {
     }
 }
 
-#[derive(Debug, Default)]
+#[derive(Clone, Debug, Default)]
```

Comment:
I noticed that the other `Report` types implement `Clone` because they're internally backed by an `Arc`, while this one actually clones the `Report` struct. This isn't a problem currently, since the system stats are collected at formatting time, but it seems like it could potentially be an issue if that behaviour changes in the future?

Reply:
Right. I think the contract is that Reports have to impl Clone. If they have they state internally, they're responsible for making it shared and internally mutable.